arch/cortex-m0/src/lib.rs

```diff
@@ -92,6 +92,13 @@ core::arch::global_asm!(
     cmp lr, r0
     bne 100f
 
+    /* We need to make sure the kernel cotinues the execution after this ISR */
+    movs r0, #0
+    msr CONTROL, r0
+    /* CONTROL writes must be followed by ISB */
+    /* http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dai0321a/BIHFJCAC.html */
```

Comment:
This link doesn't work for me. When I try to follow it, it just dumps me in the main ARM documentation page.

Reply:
Same problem for the other two copies of this link below.

Reply:
@alexandruradovici Can you fix this? I'd like to get this PR in before tagging a release candidate.

Reply:
This seems to be an alternative
```suggestion
    /* https://developer.arm.com/documentation/dui0662/b/The-Cortex-M0--Processor/Programmers-model/Core-registers */
```

Reply:
@alexandruradovici can you hit commit on this change so we can merge this PR?